docs/source/onedal/appendix/decision-tree.rst

```diff
@@ -0,0 +1,60 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _dt:
+
+Decision Tree
+*************
+
+.. toctree::
+   :maxdepth: 1
+
+
+Decision trees partition the feature space into a set of hypercubes,
+and then fit a simple model in each hypercube. The simple model can
+be a prediction model, which ignores all predictors and predicts the
+majority (most frequent) class (or the mean of a dependent variable
+for regression), also known as 0-R or constant classifier.
```

Comment:
we need to simplify this sentence

read other comments -- I suggest to have a "related terms" section, you can create a term for simple model there, and define it there in all the  details. Then here you'll have a short mention of a simple model and what it does + have a link to the detailed explanation

Reply:
The simple model is not a term in this context. 0-R is the term.